docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,354 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "116af785-a9b7-4eaf-8a86-b1eeb235da18",
+   "metadata": {},
+   "source": [
+    "# Fractional gates\n",
+    "\n",
+    "This page introduces two newly supported gate types on the IBM Quantum fleet of QPUs. These *fractional* gates are supported on [Heron QPUs](/guides/processor-types#heron) in the form of:\n",
+    "- $R_{ZZ}(\\theta)$ for $0 \\lt \\theta \\leq \\pi/2$\n",
+    "- $R_X(\\theta)$ for $0 \\lt \\theta \\leq 2\\pi$\n",
+    "\n",
+    "This page discusses the usecases which enable your quantum workflows to become more efficient using these fractional gates, as well as how to use them on IBM Quantum QPUs."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee241e99-d095-4cb5-9506-da3d92c32e1c",
+   "metadata": {},
+   "source": [
+    "## How to use fractional gates\n",
+    "\n",
+    "Internally, these fractional gates work by directly executing an $R_{ZZ}(\\theta)$ and $R_X(\\theta)$ rotation for an arbitrary angle. For the $R_X(\\theta)$ gate, this reduces the duration and error for single-qubit rotations by up to a factor of two. For the $R_{ZZ}(\\theta)$ gate, the direct execution of this two-qubit rotation avoids decomposition into multiple `CZGate`, similarly reducing a circuit's duration and error. This is especially useful for circuits which contain many single and two-qubit rotations, such as simulating the dynamics of a quantum system or when using a variational ansatz with many parameters.\n",
```

Comment:
I'm not sure, I'm not a native English speaker, but I think it should be
```
executing $R_{ZZ}(\\theta)$ and $R_X(\\theta)$ rotations for arbitrary angles
```

Reply:
This could also work:

    executing an $R_{ZZ}(\\theta)$ or an $R_X(\\theta)$ rotation for an arbitrary angle

Reply:
@abbycross What do you think?